packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -16,10 +16,17 @@ Explore the Administrator Roles API:  [![Run in Postman](https://run.pstmn.io/bu
 
 ## Role assignment operations
 
-### Grant third-party admin status
+#### Grant third-party admin status to a User
+
 <ApiOperation method="post" url="/api/v1/users/${userId}/roles?disableNotifications=true" />
 
-You can grant third-party admin status when you are adding a new admin using the API. You can do this by using an optional query parameter on the Administrator Roles API called `disableNotifications`.
+#### Grant third-party admin status to a Group
+
+<ApiOperation method="post" url="/api/v1/groups/${groupId}/roles?disableNotifications=true" />
+
+You can grant third-party admin status by using an optional query parameter on the Administrator Roles API called `disableNotifications`.
+
+You're able to grant third-party admin status when adding a new admin, or you could update their status alone by passing just the request parameter.
```

Comment:
You're able to grant > You can grant
or you could > or you can
their > its
alone > maybe change it to "individually"???

Reply:
updating to: You're able to grant third-party admin status when you assign a new role, or you can update an existing role assignment status by passing just the query parameter.